packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -321,6 +321,19 @@ const ApplicationNavigation = ({
     }
   });
 
+  useEffect(() => {
+    const forceCloseMenu = () => {
+      setDrawerMenuIsOpen(false);
+      setPopupMenuIsOpen(false);
+    };
+
+    window.addEventListener('terra-application-navigation.dismiss-menu', forceCloseMenu);
+
+    return () => {
+      window.removeEventListener('terra-application-navigation.dismiss-menu', forceCloseMenu);
```

Comment:
nitpick - we can save the event name as a constant then use that to setup and tear down the event listener

Reply:
renamed here https://github.com/cerner/terra-framework/pull/1255/commits/9c1db0419e7aa76746431034d951a205600d5b6d